### ISSUE TITLE
UserData

### ISSUE THREAD

**Issue by maustin75** [Vycka/LoadRunner#1]
I'm having an issue getting user data when inside ExecuteScenario. The testContext.UserData is null. 

```
var param = new LoadRunnerParameters {
				Limits = new ExecutionLimits {
					MaxDuration = TimeSpan.FromSeconds( 60 ),
					MaxIterationsCount = Int32.MaxValue,
					FinishTimeout = TimeSpan.FromSeconds( 120 )
				},
				SpeedStrategy = new FixedSpeed( maxIterationsPerSec: Double.MaxValue ),
				ThreadingStrategy = new IncrementalThreadCount( 15, TimeSpan.FromSeconds( 60 ), 0 ),
				InitialUserData = GetBearerToken()
			};

			var loadRunner = LoadRunnerEngine.Create<ApiTestScenario>( param, histogramAggregator );```

**Comment by Vycka** [comment]
Hey, thanks for your inquiry :)
It's a known bug, I will fix it within this weekend (or even today after my work day).

Or if by any chance you tried to fix it your self, I could merge the changes :)

**Comment by Vycka** [comment]
Should be fixed in 7.31

**Comment by maustin75** [comment]
THANKS YOU!
